ocw/lib/azure.py

```diff
@@ -137,13 +136,13 @@ def parse_image_name(self, img_name):
                        -
                        (?P<kiwi>\d+\.\d+\.\d+)
                        -
-                       (?P<flavor>Azure[-\w]*)
+                       (?P<flavor>Azure[-\w]*)?
```

Comment:
We did same change in regex for triggering for HPC , so we need to do the same in cleanup logic , right ? We need to think how to bind this logic together ...

Reply:
Good catch. But you need to include the - as optional